src/DynamicDiscreteSamplers.jl

```diff
@@ -127,7 +127,7 @@ TODO
 # 16 unused bits
 # 10236..10491           level allocated length::[UInt8 2046] (2^(x-1) is implied)
 
-# 10492..10491+2len      edit_map (maps index to current location in sub_weights)::[pos::Int, exponent::UInt64] (zero means zero; fixed location, always at the start. Force full realloc when it OOMs. TODO for perf: exponent could be UInt11, lots of wasted bits)
+# 10492..10491+len      edit_map (maps index to current location in sub_weights)::[(pos + exponent)::UInt64] (zero means zero; fixed location, always at the start. Force full realloc when it OOMs. TODO for perf: exponent could be UInt11, lots of wasted bits)
```

Comment:
This indicates that `pos + exponent` is what's stored when IIUC, it's really `pos<<11 + exponent`